### PR TITLE
Remove events to only support metrics.

### PR DESCRIPTION
**Pull request by hexedpackets**
Removes events completely from the pipeline with the exception of the riemann input.

#130

### CODE REVIEW

File: core/src/main/java/com/spotify/ffwd/debug/NettyDebugServer.java [hunk 1]
```diff
@@ -132,38 +119,32 @@ private void sendInspectPacket(Object event) throws Exception {
...
                 connected.add(ch);
                 log.info("Connected {}", ch);
 
-                ch.closeFuture().addListener(new ChannelFutureListener() {
-                    @Override
-                    public void operationComplete(ChannelFuture future) throws Exception {
-                        connected.remove(ch);
-                        log.info("Disconnected {}", ch);
-                    }
+                ch.closeFuture().addListener((ChannelFutureListener) future1 -> {
```
Comment (dmichel1) [inline core/src/main/java/com/spotify/ffwd/debug/NettyDebugServer.java]: call this `f` or `future`? you use `f` below.

File: modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java [hunk 2]
```diff
@@ -145,18 +141,4 @@ private Object decodeMetric0(final Protocol0.Metric metric) {
 
         return attributes;
     }
-
-    private Object decodeEvent0(final Protocol0.Event event) {
```
Comment (dmichel1) [inline modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java:149]: ah we have to remove event from https://github.com/spotify/ffwd-client-java too

**Review by dmichel1 — APPROVED** [review]
1 nit. LGTM after discussing more in https://github.com/spotify/ffwd/issues/130

File: core/src/main/java/com/spotify/ffwd/debug/NettyDebugServer.java (hunk 1 above)
Comment (hexedpackets) [reply core/src/main/java/com/spotify/ffwd/debug/NettyDebugServer.java]: Renamed it to `f` (this was intellij's auto-refactoring btw).

File: modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java (hunk 2 above)
Comment (hexedpackets) [reply modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java:149]: Created https://github.com/spotify/ffwd-client-java/issues/3 to do that

**Pull request merged**